src/code/z_skelanime.c

```diff
@@ -152,6 +152,7 @@ void SkelAnime_DrawFlexLimbLod(PlayState* play, s32 limbIndex, void** skeleton,
             }
             (*mtx)++;
         } else if (limbDList != NULL) {
+            if (1) {}
             MATRIX_TO_MTX(*mtx, "../z_skelanime.c", 954);
```

Comment:
startin to notice these alot, does the macro itself check if(OOT_DEBUG) 🤔 
something to look into

Reply:
I think our graphics macros might be wrong, I've noticed these have been showing up a lot in graphics functions in particular